examples/slack_webhook.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3.6		            '
+"""
+Sample config.ini
+[Slack]
+webhook = https://hooks.slack.com/services/TXXXXX/BXXXXX
+"""
+import configparser
+import json
+import sys
+
+from slack_webhook import Slack
+
+
+def main():
+    # leaving this so cron picks up this output as doublecheck
+    print('repo:', sys.argv[1])
+    print(json.dumps(json.loads(sys.argv[2]), indent=2, sort_keys=True))
+
+    # pull Slack webhook out of config file
+    # Use ConfigParser to build necessary secrets for integrations
+    config = configparser.ConfigParser()
+    config.read('config.ini')
+    slack = Slack(url=config['Slack']['webhook'])
+
+    # build variables for this wacky json
+    # there'll only ever be one repo
+    repo = sys.argv[1]
+    payload = json.loads(sys.argv[2])
+
+    filepaths = list(payload.keys())
+    for filepath in filepaths:
+        instances = payload[filepath]
+        for instance in instances:
```

Comment:
Nit: Maybe `potential_secret` instead of `instance`, just so we use the existing terminology.

Reply:
fixed terminology in https://github.com/Yelp/detect-secrets-server/pull/48/commits/45b2ccdad8bdb7cbcef9d3a3126be01b4524b04a